README.md

```diff
@@ -1,4 +1,28 @@
-# BASC QA Tool
+# Quality Install Tool
+The Quality Install Tool is a web app that implements quality installation documentation requirements for various residential energy efficiency upgrades developed by Pacific Northwest National Laboratory (PNNL) as described at http://pnnl.gov/projects/quality-install-tool. Importantly, the requirements list a set of photos that must be taken before, during, and/or after installation.
+
+### Current Version: v1.0
+
+### Features:
+- Runs on any device with a modern browser
```

Comment:
do we need to define "modern" here a bit

Reply:
It would be good to add a section with device/browser combinations that have been tested.